tests/testthat/test_civis_ml.R

```diff
@@ -516,14 +519,15 @@ test_that("passes a manifest file_id", {
   expect_args(fake_create_and_run_pred, 1,
               train_job_id = fake_model$job$id,
               train_run_id = fake_model$run$id,
+              template_id = fake_model$job$fromTemplateId + 1,
               primary_key = NULL,
               output_table = NULL,
               output_db_id = NULL,
               if_output_exists = 'fail',
               model_name = "model_task",
               n_jobs = NULL,
-              cpu_requested= NULL,
-              memory_requested= NULL,
+              cpu_requested = NULL,
+              memory_requested = NULL,
```

Comment:
Oops, that might have been my fault. Thanks for linting!

Reply:
probably Bill